src/frontend/scss/settings/_variables.scss

```diff
@@ -10,24 +10,24 @@ $onepixel: 0.0625rem;
 // Add a new breakpoint "xxl" for more granular downgrading
 $grid-breakpoints: (
   xs: 0,
-  sm: 576px,
-  md: 768px,
-  lg: 992px,
-  xl: 1200px,
-  xxl: 1900px,
+  sm: em-calc(576px),
+  md: em-calc(768px),
+  lg: em-calc(992px),
+  xl: em-calc(1200px),
+  xxl: em-calc(1900px),
```

Comment:
I'm actually used to using `em` for media queries instead of `rem` since a while now, the reason was mostly bugs in Safari (see https://zellwk.com/blog/media-query-units/ for example). I must confess I didn't check recently if there were still issues between `em` and `rem`, so that's why I stick with `em` because I'm sure it works everywhere.